common/djangoapps/student/models.py

```diff
@@ -1147,7 +1147,8 @@ def course_id(self, value):
     MODE_CACHE_NAMESPACE = u'CourseEnrollment.mode_and_active'
 
     class Meta(object):
-        unique_together = (('user', 'course'),)
+        unique_together = (('user', 'course'), )
+        index_together = (('user', 'created'),)
```

Comment:
@ormsbee while using -created I was getting following error index_together refers to the non-existent field '-created'. So that's why I had used created

Reply:
Huh. Could you try using the newer style `Index` method? https://docs.djangoproject.com/en/1.11/ref/models/indexes/#django.db.models.Index.fields